editor/components/inserter/test/menu.js

```diff
@@ -90,256 +91,240 @@ const items = [
 	sharedItem,
 ];
 
-describe( 'InserterMenu', () => {
-	// NOTE: Due to https://github.com/airbnb/enzyme/issues/1174, some of the selectors passed through to
-	// wrapper.find have had to be strengthened (and the filterWhere strengthened also), otherwise two
-	// results would be returned even though only one was in the DOM.
+const defaultProps = {
```

Comment:
This would fall under `SCREAMING_SNAKE_CASE` constant convention:

https://github.com/WordPress/gutenberg/blob/master/docs/reference/coding-guidelines.md#constants